tests/specs/SchemaFuzzSpec.elm

```diff
@@ -30,6 +30,7 @@ spec =
             , array [ items (string []) ]
             , array [ items (string []), minItems 5 ]
             , array [ items (string []), maxItems 10 ]
+            , array [ items (string []), uniqueItems False ] -- TODO: update array fuzzer
```

Comment:
Yeah, that seems hard. I think it's okay to punt on fuzzing unique items for now.